lib/elixir_conf_africa_web/live/home_live/index.html.heex

```diff
@@ -57,10 +57,18 @@
     </div>
   </div>
   <div class="w-[90%] poppins-regular  flex md:gap-8 gap-4 items-center mx-auto">
-    <p>
+    <p
+      phx-value-page_state="details"
+      phx-click="change_page_state"
+      class={"cursor-pointer #{if @page_state == "details", do: " border-b-[2px] border-[#AD3989]", else: "text-[#4D4D4D]" }"}
+    >
       Details
     </p>
-    <p>
+    <p
+      phx-value-page_state="cart"
+      phx-click="change_page_state"
```

Comment:
what is this event meant to do? I suggest you name it differently because its a bit confusing